pkg/network/errors/errors.go

```diff
@@ -3,11 +3,16 @@ package errors
 import "fmt"
 
 type CriticalNetworkError struct {
-	Msg string
+	prevErr error
```

Comment:
`prevErr` makes my skin itch (why would it be previous ? I just don't see it). 

Isn't this the wrapped error ?

Reply:
Now that you mention it, I agree.
I will rename it to `wrappedErr` as the `fmt` package names it (when using `%w`). Are you ok with that?

Reply:
Done

Reply:
> Now that you mention it, I agree. I will rename it to `wrappedErr` as the `fmt` package names it (when using `%w`). Are you ok with that?

Perfectly fine.

Thanks.